File: src/examples/tools/aam2pqr.cpp
// Converts an AAM file to PQR (for VMD)
#include "faunus/group.h"
#include "faunus/io.h"

using namespace Faunus;
using namespace std;

int main() {
  cell con(100);
  con.atom.load("../../../misc/faunatoms.dat");
  macromolecule protein;
  ioaam aam(con.atom);
  iopqr pqr(con.atom);
  protein.add( con, aam.load( "../twobody-hofmeister/lysozyme-ph4.7.aam" ));
  pqr.save("out.pqr", con.p);
}

File: src/examples/tools/aam2pqr.py
#!/usr/bin/env python

import sys
import os.path
import faunus as fau

if (len(sys.argv)==3):

  infile  = sys.argv[1]
  outfile = sys.argv[2]
  if (os.path.isfile(infile)==False):
    print "Input structure does not exist."
    sys.exit(0)

  print fau.faunus_splash()
  con  = fau.cell(1000)
  prot = fau.group()
  aam  = fau.ioaam()
  pqr  = fau.iopqr()

  prot.add(con, aam.load(infile))
  pqr.save(outfile, con.p)

else:

  print "Usage: aam2pqr infile.aam outfile.pqr"
